src/starpu/norm_fiber.cc

```diff
@@ -138,8 +140,8 @@ void restore_where()
 }
 
 template<typename T>
-void submit(Index m, Index n, Index k, Index batch, Scalar alpha, Handle src,
-        Scalar beta, Handle dst, int redux)
+void submit(Index m, Index n, Index k, Index batch, Scalar alpha, Handle src1,
+        Scalar beta, Handle src2, Handle dst, int redux)
```

Comment:
dst_mode in this function is incorrectly defined. This function ALWAYS overwrites dst handle. Therefore, access mode shall be STARPU_W in all cases.